src/app/Pages/Dashboard/Home.tsx

```diff
@@ -40,18 +42,25 @@ const Home: React.FunctionComponent<{ title: string }> = ({ title }) => {
             </Title>
           </FlexItem>
           <FlexItem
-            spacer={{ default: 'spacer2xl' }}
-            align={{ default: 'alignRight' }}>
-            <img alt={'Red Hat Enterprise Linux for the Open Hybrid Cloud'} src={logo} style={{ height: 200 }} />
+            spacer={{default: 'spacer2xl'}}
+            align={{default: 'alignRight'}}>
+            <img alt={'Red Hat Enterprise Linux for the Open Hybrid Cloud'} src={logo}
+                 style={{height: 200}}/>
           </FlexItem>
         </Flex>
       </PageSection>
       <PageSection variant={PageSectionVariants.light}>
-        <ImageTable />
+        <Flex alignContent={ {default: "alignContentStretch" }} grow={{ default: 'grow' }} display={{ default: 'inlineFlex' }}>
```

Comment:
Needs some consistency (space or not) for the curly brackets.

Reply:
I would recommend getting Prettier set up to handle this automatically.

Reply:
I use prettier as well. I think it would make sense if we use the same linter.

Reply:
Will do. Thanks =)